lib/omniauth/strategies/openid_connect.rb

```diff
@@ -222,7 +255,8 @@ def access_token
 
         @access_token = client.access_token!(
           scope: (options.scope if options.send_scope_to_token_endpoint),
-          client_auth_method: options.client_auth_method
+          client_auth_method: options.client_auth_method,
+          code_verifier: (session.delete('omniauth.pkce.verifier') if options.pkce)
```

Comment:
Is this query parameter is omitted if PKCE is not  used? We might want to omit it entirely so we don't introduce confusion or a bug.